.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,24 +1,31 @@
 ---
-name: 🐛 Bug Report
-about: If something isn't working as expected 🤔.
+name: 🐛 Bug report
+about: If something isn't working as expected 🤔
+labels: bug
 
 ---
 
-## Bug Report
+## Bug report
 
-### Steps to Reproduce:
- 1. ...step 1 description...
- 2. ...step 2 description...
- 3. ...step 3 description...
+Be sure to search existing issues since someone might have already raised this bug already.
```

Comment:
I'd consider dropping this text, it is an additional obstacle - I'm guessing you don't have too many issues with people adding duplicates yet?